guides/hosting/installation-updates/deployments/build-w-o-db.md

```diff
@@ -24,9 +26,805 @@ storefront:
     theme:
         config_loader_id: Shopware\Storefront\Theme\ConfigLoader\StaticFileConfigLoader
         available_theme_provider: Shopware\Storefront\Theme\ConfigLoader\StaticFileAvailableThemeProvider
+        theme_path_builder_id: Shopware\Storefront\Theme\MD5ThemePathBuilder
  ```
 
- This will force the theme compiler to use the static dumped file instead of looking into the database.
+This will force the theme compiler to use the static dumped file instead of looking into the database.
+
+{% hint style="information" %}
+Warnings about Database errors can still occur but will be catched and should be ignored in this case.
```

Comment:
@ssltg, Are you referring to the error being cached or caught?

Reply:
The errors being caught. Should I write "caught" instead of "catched"?

Reply:
I answerd it by myself. Sure it should be "caught" :-D

Reply:
Its changed now :)